product_supplierinfo_import/wizards/product_supplierinfo_import.py

```diff
@@ -16,17 +17,42 @@ class ProductSupplierInfoImport(models.TransientModel):
     _name = "product.supplierinfo.import"
     _description = "Import supplier info records"
 
-    supplier_id = fields.Many2one(comodel_name="res.partner", required=True)
+    supplier_id = fields.Many2one(
+        "res.partner",
```

Comment:
This is a formal preference, but we prefer to be explicit for better readeability
```suggestion
        comodel_name="res.partner",